plugin.yaml

```diff
@@ -5,6 +5,6 @@ version: "2.8.2+2"
 usage: "Preview helm upgrade changes as a diff"
 description: "Preview helm upgrade changes as a diff"
 useTunnel: true
-command: "$HELM_PLUGIN_DIR/bin/diff"
+command: "$HELM_PLUGIN_DIR/diff"
```

Comment:
this conflicts with master and needs to be reverted.

Reply:
resolved it